inc/taxonomies.php

```diff
@@ -134,15 +139,22 @@ function largo_custom_taxonomies() {
 			array(
 				'label' => __( 'Post Types', 'largo' ),
 				'labels' => array(
-					'name' => __( 'Post Types', 'largo' ),
-					'singular_name' => __( 'Post Type', 'largo' ),
-					'all_items' => __( 'All Post Types', 'largo' ),
-					'edit_item' => __( 'Edit Post Type', 'largo' ),
-					'update_item' => __( 'Update Post Type', 'largo' ),
-					'view_item' => __( 'View Post Type', 'largo' ),
-					'add_new_item' => __( 'Add New Post Type', 'largo' ),
-					'new_item_name' => __( 'New Post Type Name', 'largo' ),
-					'search_items' => __( 'Search Post Type'),
+					'name'                  => __( 'Post Types', 'largo' ),
+					'singular_name'         => __( 'Post Type', 'largo' ),
```

Comment:
@benlk Do we want `name` and `singular_name` here to allow for specifying a context for translation like in the `Post Prominence` and `Series` taxonomies?

Reply:
yes, probably something like "taxonomy label".